packages/core/src/lib/jdk/JarSigner.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import {JdkHelper} from './JdkHelper';
+import {executeFile} from '../util';
+import {SigningKeyInfo} from '../../lib/TwaManifest';
+
+const JARSIGNER_CMD = 'jarsigner';
+const SIGALG = 'SHA256withRSA';
+const DIGESTALG = 'SHA-256';
```

Comment:
I know you've just copied the names from the flags `jarsigner` has, but do you mind making them more descriptive - `SIGNATURE_ALGORITHM` and `DIGEST_ALGORITHM`?

Reply:
done!